src/app/page/page.component.ts

```diff
@@ -24,7 +24,14 @@ namespace Page {
           this.previousWidth = this.$window.innerWidth;
         });
       });
-      this.$rootScope.$on(CLOSE_MAIN_NAV_EVENT, () => this.isNavOpen = false);
+      let curr_state = window.localStorage.getItem('defaultVerticalNavState');
+      if (curr_state === "show") {
+        this.isNavOpen = true;
+      }
```

Comment:
no line break here

Reply:
Still not resolved yet.

Reply:
should i change the '}' to line 29

Reply:
![Screenshot from 2022-02-24 09-31-35](https://user-images.githubusercontent.com/49123028/155455828-bc52d99e-0e53-4aae-9a78-92d3dc125400.png)
This is how it looks in my vs-code